src/plone/restapi/services/copymove/copymove.py

```diff
@@ -1,12 +1,17 @@
 # -*- coding: utf-8 -*-
 from Acquisition import aq_parent
-from Products.CMFCore.utils import getToolByName
+from plone import api
```

Comment:
@sneridagh we haven't made a final decision regarding plone.api usage in plone.restapi. If it is not too much work I would remove the dependency here. Keep them in tests is fine though. The alternative would be to get a quick feedback from the @plone/framework-team about this. Though, I guess it would take some time do discuss the details.